cmd/ocm/cluster/create/create.go

```diff
@@ -164,8 +201,13 @@ func run(cmd *cobra.Command, argv []string) error {
 		Version(
 			cmv1.NewVersion().
 				ID(clusterVersion),
-		).
-		Build()
+		)
+	if args.expiration != nil {
+		clusterBuild = clusterBuild.ExpirationTimestamp(
+			args.expiration.Time,
```

Comment:
Do you need the metav1.Time wrapping if in the end  you extract .Time field?

Reply:
I was using it for nil checking, but I can use `IsZero()` instead.